src/ops/inc_multihead_self_attention.cu

```diff
@@ -31,12 +31,13 @@ __global__ void build_w_out_tensor(float const *weight_ptr,
                                    int num_heads,
                                    int qkv_weight_block_size) {
   CUDA_KERNEL_LOOP(i, vProjSize * oProjSize * num_heads) {
-    int row_idx = i % vProjSize;
-    int col_idx = (i / vProjSize) % oProjSize;
+    int v_idx = i % vProjSize;
+    int o_idx = (i / vProjSize) % oProjSize;
     int head_idx = i / (vProjSize * oProjSize);
-    contiguous_weight_ptr[i] =
```

Comment:
@xinhaoc I think this was a bug, because the contiguous_weight_ptr has shape: [vProjSize, num_heads, out_dim] in column-major order. Will fixing this break the LLAMA alignment? Could you double check if LLAMA still works or if we need to make some fix?

Reply:
sorry I just saw it, let me check.

Reply:
yes, the result will be not aligned if changes this.